packages/react/src/global-nav-profile-menu/GlobalNavProfileMenu.tsx

```diff
@@ -0,0 +1,44 @@
+import type { ReactNode } from "react";
+
+import * as RadixCollapsible from "@radix-ui/react-collapsible";
+
+import { Box, type BoxProps } from "../box";
+import { Button } from "../button";
+import { Flex } from "../flex";
+import { IconEllipsis } from "../icons/IconEllipsis";
+import { Text } from "../text";
+
+export type GlobalNavProfileMenuProps = BoxProps<
+  "div",
+  {
+    avatar: ReactNode;
+    name?: string;
+    organization?: string;
+  }
+>;
+
+export const GlobalNavProfileMenu = ({
+  avatar,
+  name,
+  organization,
+}: GlobalNavProfileMenuProps) => {
+  return (
+    <Flex alignItems="center" flexDirection="row" gap="xs" px="md" py="xs">
```

Comment:
For the `gap` the image isn't positioned in the center. We need to set gap={0} and add a `ml` to the Flex with name and organization

Reply:
thanks!